client/app/locales/de.json

```diff
@@ -20,6 +20,7 @@
     "modal error folder create": "Ordner konnte nicht erstellt werden",
     "modal error folder exists": "Entschuldigung, eine Datei oder ein Ordner mit diesem Namen existiert bereits",
     "modal error zip empty folder": "Ein leerer Ordner kann nicht als ZIP herunter geladen werden.",
+    "modal error updoad size exceed": "For performance reasons, you cannot upload more than %{maxSize} files at the same time. Please try again with fewer files.",
```

Comment:
I don't think we should not update non-en locale without using Transifex. Furthermore, you have put the english translations here in the German locale.